forms-flow-api/src/formsflow_api/resources/employeeData.py

```diff
@@ -65,3 +70,31 @@ def get():
             return err.error, err.status_code
 
         return employee_names, HTTPStatus.OK
+
+
+@cors_preflight("POST, OPTIONS")
+@API.route("/user-group", methods=["POST", "OPTIONS"])
+class EmployeeGroup(Resource):
+    """Employee and Group Data"""
+
+    @staticmethod
+    @profiletime
+    @auth.require
+    @auth.has_one_of_roles([DESIGNER_GROUP])
+    def post():
+        """Process users"""
+        try:
+            data = request.get_json()
+            group = data.get("group")
+            users = data.get("users")
+            if group and users:
+                obj = KeycloakService()
+                result, groupId = obj.add_users(group, users)
+                if groupId:
+                    # print(er)
```

Comment:
Can probably remove this commented out print statement

Reply:
Fixed